jenkins-agent-v3.40/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ghcr.io/boozallen/sdp-images/jenkins-agent:3.23
```

Comment:
I'm not super familiar with the build / tagging process for sdp images. Is there any way that this image could change in a way that breaks the rest of the steps used here? Wondering if we should also pin to a certain SHA. But, again, not really familiar with this process and maybe that's more than we really need to do.

Reply:
Don't think this could easily become a problem here, but it doesn't hurt to add the SHA, so I will.

Reply:
Cool, sounds good!